### PR TITLE
Implement TrackSetBrakeSpeedAction

### PR DESCRIPTION
**Pull request by duncanspumpkin**

### CODE REVIEW

File: src/openrct2/actions/GameActionRegistration.cpp [hunk 1]
```diff
@@ -51,6 +51,7 @@
 #include "TrackRemoveAction.hpp"
 #include "WallRemoveAction.hpp"
 #include "WaterSetHeightAction.hpp"
+#include "TrackSetBrakeSpeedAction.hpp"
```
Comment (ZehMatt) [inline src/openrct2/actions/GameActionRegistration.cpp]: Gotta also register the action.

**Comment by Gymnasiast** [comment]
Have you also tested if setting booster speeds still works?

**Comment by duncanspumpkin** [comment]
This action is untested at present.

**Comment by duncanspumpkin** [comment]
Part of #8878

**Pull request closed without merging**